aces/Loris
# [Behavioural QC] updated module help text

New copy for the Behavioural QC module help text, formerly the **Data Team Helper**

#### Testing instructions

1. Review copy (again, I paired this one down a lot so make sure I haven't left out anything vital)
2. see screenshot of rendering

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

![Screenshot 2019-11-14 15 43 32](https://user-images.githubusercontent.com/37813606/68894576-8c5ed200-06f5-11ea-92e5-becbd463d03a.png)
